bot.rb

```diff
@@ -109,9 +109,13 @@ def process_open_issue(issue)
       bot_actions = []
       process_inactive(issue)
 
-      return if issue.comments > 0 # there maybe already some bot replys
+      # return if issue.comments > 0 # there maybe already some bot replies
       bot_actions << process_code_signing(issue)
-      bot_actions << process_env_check(issue)
+      # bot_actions << process_env_check(issue)
```

Comment:
Let's remove these lines if we don't need them any more

Reply:
Sorry, forgot to uncomment it. I commented out for easier testing.